python/mpt/src/mpt/trie_diff.py

```diff
@@ -386,3 +378,15 @@ def _process_storage_diff(
             if address not in self._storage_tries:
                 self._storage_tries[address] = {}
             self._storage_tries[address][key] = tuple((left_decoded, right_decoded))
+
+
+def resolve(
```

Comment:
i dont think resolve is used with internal node as argument

Reply:
it is